src/static/vimtutor.txt

```diff
@@ -0,0 +1,808 @@
+===============================================================================
+=    W e l c o m e   t o   t h e   V I M   T u t o r    -    Version 1.5      =
```

Comment:
Have you made any edits to this, or is this straight from `vimtutor`?

Reply:
directly from here: https://github.com/microsoft/vscode-extension-samples/blob/master/vim-sample/vimtutor.txt